thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/lib/FunctionReplayRunnable.java

```diff
@@ -37,6 +37,7 @@
   private Long functionAutotuneConfigId;
   private boolean speedUp;
   private boolean selfKill;
+  private long clonedFunctionId;
```

Comment:
is these runnable going to create multiple cloneFunctionIds, if it takes multiple configurations? So you will always return the last one, is this the expected behavior?